main.css

```diff
@@ -1,3 +1,12 @@
+html,
+body {
+  font-family: "Roboto", sans-serif;
+  width: 100%;
+  margin: 0;
+  padding: 0;
+  font-size: 16px; /* for rem */
+}
+
```

Comment:
@takshch Please remove these whitespaces after each CSS block.

Reply:
@RohitKuwar Can you please tell me why it is better to remove whitespaces or new line after each CSS block.

Reply:
I agree with Taksh here. That's how I do it as well.